components/src/LiveCode/index.tsx

```diff
@@ -0,0 +1,25 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui'
+import Default from './liveCodeScope'
+import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live'
+
+export const LiveCode = ({ scope = {}, code }: Props) => (
+  <LiveProvider scope={{ ...Default, scope }} code={code}>
```

Comment:
```suggestion
  <LiveProvider scope={{ ...defaultScope, ...scope }} code={code}>
```
It was missing the spread operator (`...scope`), so it was placing the object inside the other instead of distributing its properties to the `LiveProvider` scope :detective: